packages/stryker/src/Stryker.ts

```diff
@@ -7,57 +8,60 @@ import ReporterOrchestrator from './ReporterOrchestrator';
 import TestFrameworkOrchestrator from './TestFrameworkOrchestrator';
 import MutantTestMatcher from './MutantTestMatcher';
 import InputFileResolver from './input/InputFileResolver';
-import ConfigReader from './ConfigReader';
+import ConfigReader from './config/ConfigReader';
 import PluginLoader from './PluginLoader';
 import ScoreResultCalculator from './ScoreResultCalculator';
-import ConfigValidator from './ConfigValidator';
+import ConfigValidator from './config/ConfigValidator';
 import { freezeRecursively, isPromise } from './utils/objectUtils';
 import { TempFolder } from './utils/TempFolder';
-import * as log4js from 'log4js';
 import Timer from './utils/Timer';
 import StrictReporter from './reporters/StrictReporter';
 import MutatorFacade from './MutatorFacade';
 import InitialTestExecutor, { InitialTestRunResult } from './process/InitialTestExecutor';
 import MutationTestExecutor from './process/MutationTestExecutor';
 import InputFileCollection from './input/InputFileCollection';
+import LogConfigurator from './logging/LogConfigurator';
 
 export default class Stryker {
 
   config: Config;
   private timer = new Timer();
   private reporter: StrictReporter;
   private testFramework: TestFramework | null;
-  private readonly log = log4js.getLogger(Stryker.name);
+  private readonly log: Logger;
 
   /**
    * The Stryker mutation tester.
    * @constructor
    * @param {Object} [options] - Optional options.
    */
   constructor(options: StrykerOptions) {
+    LogConfigurator.forMaster(options.logLevel, options.fileLogLevel);
```

Comment:
I don't know what this line does if I just read the function call

Reply:
Renamed it to `configureMainStrykerProcess`